src/collection.js

```diff
@@ -158,24 +160,26 @@ export default class Collection {
    * Adds a record to the local database.
    *
    * Options:
-   * - {Boolean} synced     Sets record status to "synced" (default: false).
-   * - {Boolean} forceUUID  Enforces record creation using any provided UUID
-   *                        (default: false).
+   * - {Boolean} synced   Sets record status to "synced" (default: false).
+   * - {Boolean} forceId  Forces the id field from the record to be used, instead
+   *                      of one that is generated automatically (default: false).
```

Comment:
OK, I started doing this but it requires some code restructuring and extra tests, so I'll make it a follow-up ticket if that's OK (it's a good idea anyway to check for this, regardless of whether you use the default or a custom IdSchema).